geth-dev-node/genesis.json

```diff
@@ -1,17 +1,798 @@
+    },
+    "00000000000000000000000000000000000000dc": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000dd": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000de": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000df": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e0": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e1": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e2": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e3": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e4": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e5": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e6": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e7": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e8": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e9": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ea": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000eb": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ec": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ed": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ee": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ef": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f0": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f1": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f2": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f3": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f4": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f5": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f6": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f7": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f8": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f9": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fa": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fb": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fc": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fd": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fe": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ff": {
+      "balance": "0x1"
+    },
```

Comment:
Consider doing a find-and-replace and making all of these accounts one-line rather than 3 lines.  This would just be to shrink the file a bit and make it easier to read.

Also, do we really need 0xff worth of these accounts with a balance of 1?  I suspect we could delete all but the first 10-20 and be fine.

Reply:
Yeah, this was how puppeth generated the file.  I wasn't sure if these accounts were for anything in particular, and that's how one of the clique files you gave me looked, so I left them in.  I'll remove some of them and reformat these lines.